doc/lua_api.md

```diff
@@ -9005,33 +9014,41 @@ offering very strong randomness.
 * `get_state()`: return generator state encoded in string
 * `set_state(state_string)`: restore generator state from encoded string
 
-`PerlinNoise`
+`ValueNoise`
 -------------
 
-A perlin noise generator.
-It can be created via `PerlinNoise()` or `core.get_perlin()`.
-For `core.get_perlin()`, the actual seed used is the noiseparams seed
+A value noise generator.
+It can be created via `ValueNoise()` or `core.get_value_noise()`.
+For legacy reasons, it can also be created via `PerlinNoise()` or `core.get_perlin()`,
+but the implemented noise is not Perlin noise.
+For `core.get_value_noise()`, the actual seed used is the noiseparams seed
 plus the world seed, to create world-specific noise.
 
-`PerlinNoise(noiseparams)`
+`ValueNoise(noiseparams)`
```

Comment:
can we format this as a list? otherwise with how markdown works it'll all be squeezed into one paragraph.